### ISSUE TITLE
print failure

### ISSUE THREAD

**Issue by molmodel** [daid/LegacyCura#267]
In repeating a 10-hr print, the program 'froze' during the eighth hour.
All print conditions were identical to a previous, successful print.
Specifically, I was running a Dell pc, not hooked up to internet, and had a surge protector but not with uninterruptible power.
It would appear that the Ultimaker printer was functioning normally.
Have others had a similar experience?

Thanks,
EM

**Comment by daid** [comment]
I've seen it before (rarely) so far I've only traced it down to the USB drivers giving up and causing mayhem. I highly recommend the UltiController upgrade (home build or bought) and printing trough SD, as this is 100% stable compared to USB printing.

**Comment by molmodel** [comment]
Daid,thabnks for the infomight an uninterruptible power supply solve the problem?Edgar

http://molecular-sculpture.com

--- On Wed, 11/21/12, daid notifications@github.com wrote:

**Comment by molmodel** [comment]
Daid,
here are the symptoms of a 24hr print job freezing after 20hrs:
The print process hangs, with power on and the extruder tip remaining hot
but the motors have lost power, as I can easily move them by hand
The interface panel does not respond to mouse command, like "pause", but, gives a response "M105 ..." when I click on the "check temperature" button - but does not print out the temperature...
because of the hung interface, I cannot restart or continue printing and have to kill the job.

separately, do you have a version of Cura that will run on iMac OS 10.4.11 ?
thanks,
edgar

**Comment by daid** [comment]
Sounds a lot like the USB-Serial driver hanging, which can happen, nothing I can do about that really. (It would even be impossible to recover after detecting a case like this)

Cura is officially supported on Mac 10.6 or higher, and I have heard reports about people running it on 10.5 (with some effort) But 10.4 is quite old in that aspect, and if you are running on PPC, forget it. In all cases, below 10.6, you are on your own.

**Comment by molmodel** [comment]
Daid, I've attached a photo showing a translation in the y direction of ca. 5mm towards the end of the print-I adjusted temperature (220 to 205C) and speed (100% to 60% of 50mm/s) at about this timethis is the second time I have noticed a mid-print translation shift.Is this common?
And, as I have your eye, let me tell you how delighted I am with the composition program in Cura 12.11running on my iMac - it comes very close to solving my problem of printing the undersurfaces of hanging spheres (aka 'atoms') in my sculptures - an elegant solution - thanksbut, I am unable to run the print program of Cura 12.11 on my PC - a black screen flashes and disappears, so I am running 12.08 - is there some fix?
Edgar

http://molecular-sculpture.com

--- On Wed, 11/28/12, daid notifications@github.com wrote:
